### ISSUE TITLE
How to import breakpoint with webpack?

### ISSUE THREAD

**Issue by vdzundza** [at-import/breakpoint#167]
Hi guys,

I've installed susy and sass, after that include css/sass loader to webpack config

`{
        test: /\.scss$/,
        loader: ExtractTextPlugin.extract('css!sass')
  }`

Here is my main scss file

```
@import "~susy/sass/susy";
@import "breakpoint";

$susy: (
  columns: 12,
  gutters: 1/4,
  math: fluid,
  output: float,
  gutter-position: inside
);

.layout {
  @include container();
  @include layout(12 1/4);
}
```

This is an error in a webpack output

```
ERROR in ./src/styles/base.scss
Module build failed: ModuleBuildError: Module build failed: 
@import "breakpoint";
^
      File to import not found or unreadable: breakpoint
```

Does anyone know how to properly import breakpoint ?

**Comment by ShaunKT** [comment]
Have a look at http://hownodejs.com/39/01/how-to-import-ruby-gems-breakpoint-with-webpack.html it seems to have a solution

**Comment by ignaciosegura** [comment]
Assuming you installed breakpoint-sass package:

`npm install breakpoint-sass --save-dev`

Then you have to import it letting the SASS loader know that this element you want to import, that is, "breakpoint", is outside the SASS folder, with the other node modules:

`@import '~breakpoint-sass/stylesheets/breakpoint';`

**Comment by tavurth** [comment]
Perhaps we could add this to the Readme, or rather the wiki?

**Comment by galakhov** [comment]
Yes, please add this part to the installation to the project's wiki:
`npm i -D breakpoint-sass
@import "../node_modules/breakpoint-sass/stylesheets/_breakpoint";`

**Comment by Tamenze** [comment]
+1 on the wiki edit
